<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HtmlForm</class>
 <widget class="QWidget" name="HtmlForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>600</height>
      </size>
     </property>
     <property name="title">
      <string>Blocks</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="block1Button">
        <property name="maximumSize">
         <size>
          <width>220</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="image.qrc">
          <normaloff>:/icons/block_1@2x.png</normaloff>:/icons/block_1@2x.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>200</width>
          <height>60</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="block2Button">
        <property name="maximumSize">
         <size>
          <width>220</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="image.qrc">
          <normaloff>:/icons/block_2@2x.png</normaloff>:/icons/block_2@2x.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>200</width>
          <height>60</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="block3Button">
        <property name="maximumSize">
         <size>
          <width>220</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="image.qrc">
          <normaloff>:/icons/block_3@2x.png</normaloff>:/icons/block_3@2x.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>200</width>
          <height>60</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="block4Button">
        <property name="maximumSize">
         <size>
          <width>220</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="image.qrc">
          <normaloff>:/icons/block_4@2x.png</normaloff>:/icons/block_4@2x.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>200</width>
          <height>60</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="block5Button">
        <property name="maximumSize">
         <size>
          <width>220</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="image.qrc">
          <normaloff>:/icons/block_5@2x.png</normaloff>:/icons/block_5@2x.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>200</width>
          <height>60</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QTextEdit" name="textEdit">
     <property name="maximumSize">
      <size>
       <width>375</width>
       <height>600</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QWebView" name="webView">
     <property name="maximumSize">
      <size>
       <width>375</width>
       <height>600</height>
      </size>
     </property>
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openButton">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="proceedButton">
       <property name="text">
        <string>Proceed</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>webView</zorder>
  <zorder>textEdit</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
